matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/timeline/DefaultTimeline.kt

```diff
@@ -727,6 +728,9 @@ internal class DefaultTimeline(
             not().like(TimelineEventEntityFields.ROOT.CONTENT, FilterContent.EDIT_TYPE)
             not().like(TimelineEventEntityFields.ROOT.CONTENT, FilterContent.RESPONSE_TYPE)
         }
+        if (settings.filterRedacted) {
+            not().like(TimelineEventEntityFields.ROOT.UNSIGNED_DATA, UnsignedContent.REDACTED_TYPE)
```

Comment:
You should also handle this filter in TimelineHiddenReadReceipts class

Reply:
Yes, good point, done